guides/payments.md

```diff
@@ -0,0 +1,220 @@
+`Payment#purchase!` method will be called, which will call
+`PaymentMethod#purchase` like this:
+
+```ruby
+payment_method.purchase(<amount>, <source>, <gateway options>)
+```
+
+If the payment is *not* configured to auto-capture payments, the
+`Payment#authorize!` method will be called, with the same arguments as the
+`purchase` method above:
+
+```ruby
+payment_method.authorize(<amount>, <source>, <gateway options>)
+```
+
+How the payment is actually put through depends on the `PaymentMethod`
+sub-class' implementation of the `purchase` and `authorize` methods.
+
+The returned object from both the `purchase` and `authorize` methods on the 
+payment method objects must be an `ActiveMerchant::Billing::Response` object.
+This response object is then stored (in YAML) in the `spree_log_entries` table.
+Log entries can be retrieved with a call to the `log_entries` association on any
+`Payment` object.
+
+If the `purchase!` route is taken and is successful, the payment is marked as
+`completed`. If it fails, it is marked as `failed`. If the `authorize` method is
+successful, the payment is transitioned to the `pending` state so that it can be
+manually captured later by calling the `capture!` method. If it is unsuccessful,
+it is also transitioned to the `failed` state.
+
+***
+Once a payment has been saved, it also updates the order. This may trigger the
+`payment_state` to change, which would reflect the current payment state of the
+order. The possible states are:
+
+* `balance_due`: Indicates that payment is required for this order
+* `failed`: Indicates that the last payment for the order failed
+* `credit_owed`: This order has been paid for in excess of its total
+* `paid`: This order has been paid for in full.
+***
+
+!!!
+You may want to keep tabs on the number of orders with a `payment_state` of
+`failed`. A sudden increase in the number of such orders could indicate a
+problem with your credit card gateway and most likely indicates a serious
+problem affecting customer satisfaction. You should check the latest
+`log_entries` for the most recent payments in the store if this is happening.
+!!!
+
+### Log Entries
+
+Responses from payment gateways within Solidus are typically
+`ActiveMerchant::Billing::Response` objects. When Solidus handles a response
+from a payment gateway, it will serialize the object as YAML and store it in the
+database as a log entry for a payment. These responses can be useful for
+debugging why a payment has failed.
+
+You can get a list of these log entries by calling the `log_entries` on any 
+`Spree::Payment` object. To get the `Active::Merchant::Billing::Response` out of these `Spree::LogEntry` objects, call the `details` method.
+
+## Supported Gateways
+
+Access to a number of payment gateways is handled with the usage of the [solidus_gateway](https://github.com/solidusio/solidus_gateway) extension. This extension currently supports the following gateways:
```

Comment:
I just used `Stripe` from `solidus_gateway`. So, not sure if this list of supported gateways apply to `solidus_gateway` right now?